tx-pool/src/pool.rs

```diff
@@ -59,6 +62,7 @@ impl TxPool {
             recent_reject,
             expiry,
             conflicts_cache: LruCache::new(CONFLICTES_CACHE_SIZE),
+            conflicts_outputs_cache: lru::LruCache::new(CONFLICTES_INPUTS_CACHE_SIZE),
```

Comment:
we may add two config options for `CONFLICTES_CACHE_SIZE` and `CONFLICTES_INPUTS_CACHE_SIZE`